packages/material-ui/src/FormControl/FormControl.test.js

```diff
@@ -119,6 +119,14 @@ describe('<FormControl />', () => {
       container.querySelector('input').blur();
       expect(readContext.args[0][0]).to.have.property('focused', true);
     });
+
+    it('should not go in an infinite loop', () => {
+      render(
+        <FormControl focused disabled>
+          <Input />
+        </FormControl>,
+      );
+    });
```

Comment:
Could you include assertions for which element is focused and which element is disabled in this case. Since we support conflicting props we need to define what result it should produce.